pyproject.toml

```diff
@@ -44,7 +44,7 @@ click = "^8.1.3"
 opencv-python = "^4.7.0.72"
 cython = "^0.29.34"
 pyyaml = "^6.0"
-pillow = "^9.5.0"
+pillow = "10.1.0"
```

Comment:
What's the reason for setting pillow to the fixed version `10.1.0`.  I would recommend to use `^10.1.0` od `~10.1.0`